xarray/core/merge.py

```diff
@@ -513,6 +513,24 @@ def merge_attrs(variable_attrs, combine_attrs):
                     "the same. Merging %s with %s" % (str(result), str(attrs))
                 )
         return result
+    elif combine_attrs == "drop_conflicts":
+        result = {}
+        dropped_keys = set()
+        for attrs in variable_attrs:
+            result.update(
+                {
+                    key: value
+                    for key, value in attrs.items()
+                    if key not in result and key not in dropped_keys
+                }
+            )
+            result = {
+                key: value
+                for key, value in result.items()
+                if key not in attrs or equivalent(attrs[key], value)
```

Comment:
`equivalent(attrs[key], value)` 

This could fail badly for some weird attribute value (object that doesn't implement `__eq__`?). Can we guard against that?

Reply:
not sure, but this is the case for `"no_conflicts"` and `"identical"`, too: `dict_equiv` and `compat_dict_union` use `equivalent` by default.

Reply:
ah OK :+1:

Reply:
also:
```python
In [1]: object().__eq__(object())
Out[1]: NotImplemented

In [2]: object() == object()
Out[2]: False
```
so I guess that just means these objects would not be seen as equal?

Reply:
perfect!